### PR TITLE
[haste-server] Newchart haste

### PR DESCRIPTION
**Pull request by rwaltr**
**Description of the change**

New Chart!

**Benefits**

Yet Another chart

**Possible drawbacks**

There are currently two versions of Haste-server floating around k8s-at-home, there is the original 

- https://github.com/seejohnrun/haste-server 

and the version being built at 

- https://github.com/k8s-at-home/container-images/tree/main/apps/haste being built from https://github.com/zneix/haste-server

The upstream version offers ENV based config editing, which is preferable, while Zneix does not and requires a config file to be build and injected via configmap or otherwise. 

~looking for input on which we should use, upstream container does not seem to be using any odd s6 overlays~ We will use upstream

**Applicable issues**

**Additional information**

**Checklist** 
- [x] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [x] Title of the PR starts with chart name (e.g. `[home-assistant]`)
- [x] Variables are documented in the README.md (this can be done with using our helm-docs wrapper `./hack/gen-helm-docs.sh stable <chart>`)

### CODE REVIEW

File: charts/stable/haste/Chart.yaml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+apiVersion: v2
+appVersion: 2021.04.18
+description: haste helm package
```
Comment (onedr0p) [inline charts/stable/haste/Chart.yaml:3]: What's haste? 😅

File: charts/stable/haste/Chart.yaml [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+name: haste
+version: 1.1.0
```
Comment (onedr0p) [inline charts/stable/haste/Chart.yaml:5]: Should probably start at 1.0.0

File: charts/stable/haste/Chart.yaml [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+icon: https://github.com/seejohnrun/haste-server/blob/master/static/logo.png
+sources:
+- https://github.com/seejohnrun/haste-server
+maintainers:
+- name: rwaltr
+  email: rwalt@pm.me
+dependencies:
+- name: common
+  repository: https://library-charts.k8s-at-home.com
+  version: 2.2.0
```
Comment (onedr0p) [inline charts/stable/haste/Chart.yaml:22]: There's a newer common library version than this.

File: charts/stable/haste/README.md [hunk 4]
```diff
@@ -0,0 +1,116 @@
+# haste
```
Comment (onedr0p) [inline charts/stable/haste/README.md:1]: Need to rerun helm-docs again after all these changes

File: charts/stable/hastebin/chart/Chart.yaml [hunk 5]
```diff
@@ -0,0 +1,22 @@
+apiVersion: v2
+appVersion: 1.0.0
+description: haste helm package
```
Comment (onedr0p) [inline charts/stable/hastebin/chart/Chart.yaml:3]: It does what now?

File: charts/stable/hastebin/chart/Chart.yaml [hunk 6]
```diff
@@ -0,0 +1,22 @@
...
+name: haste
+version: 1.1.0
```
Comment (onedr0p) [inline charts/stable/hastebin/chart/Chart.yaml:5]: Should probably start at 1.0.0

File: charts/stable/hastebin/chart/Chart.yaml [hunk 7]
```diff
@@ -0,0 +1,22 @@
...
+icon: https://github.com/seejohnrun/haste-server/blob/master/static/logo.png
+sources:
+- https://github.com/seejohnrun/haste-server
+maintainers:
+- name: rwaltr
+  email: rwalt@pm.me
+dependencies:
+- name: common
+  repository: https://library-charts.k8s-at-home.com
+  version: 2.2.0
```
Comment (onedr0p) [inline charts/stable/hastebin/chart/Chart.yaml:22]: Update this to latest common library version

File: charts/stable/hastebin/chart/README.md [hunk 8]
```diff
@@ -0,0 +1,116 @@
+# ${CHARTNAME}
```
Comment (onedr0p) [inline charts/stable/hastebin/chart/README.md:1]: Run helm docs

File: charts/stable/hastebin/chart/values.yaml [hunk 9]
```diff
@@ -0,0 +1,33 @@
...
+# IMPORTANT NOTE
+#
+# This chart inherits from our common library chart. You can check the default values/options here:
+# https://github.com/k8s-at-home/library-charts/tree/main/charts/stable/common/values.yaml
+#
+
+image:
+  repository: ghcr.io/k8s-at-home/haste
+  pullPolicy: IfNotPresent
+  tag: vce75235
```
Comment (onedr0p) [inline charts/stable/hastebin/chart/values.yaml:11]: Does this tag exist?

File: charts/stable/hastebin/chart/values.yaml [hunk 10]
```diff
@@ -0,0 +1,33 @@
...
+
+strategy:
+  type: Recreate
+
+# See more environment variables in the ${CHARTNAME} documentation
+# https://${CHARTNAME}.org/docs
```
Comment (onedr0p) [inline charts/stable/hastebin/chart/values.yaml:17]: Fix

**Review by onedr0p — CHANGES_REQUESTED** [review]
:wave: what's up with 2 helm charts in one PR?

File: charts/stable/hastebin/chart/README.md (hunk 8 above)
Comment (rwaltr) [reply charts/stable/hastebin/chart/README.md:1]: some weirdness happened, there should not be two hastebins charts

File: charts/stable/hastebin/chart/values.yaml (hunk 10 above)
Comment (rwaltr) [reply charts/stable/hastebin/chart/values.yaml]: removed duplicate

File: charts/stable/hastebin/chart/values.yaml (hunk 9 above)
Comment (rwaltr) [reply charts/stable/hastebin/chart/values.yaml]: Use to, may not anymore,
removed duplicate

File: charts/stable/hastebin/chart/README.md (hunk 8 above)
Comment (rwaltr) [reply charts/stable/hastebin/chart/README.md]: removed duplicate

File: charts/stable/hastebin/chart/Chart.yaml (hunk 7 above)
Comment (rwaltr) [reply charts/stable/hastebin/chart/Chart.yaml]: updated in https://github.com/k8s-at-home/charts/pull/800/commits/769f44f54a9f96ea280834e666a3c0e2d46314b0

File: charts/stable/hastebin/chart/Chart.yaml (hunk 5 above)
Comment (rwaltr) [reply charts/stable/hastebin/chart/Chart.yaml]: updated

File: charts/stable/haste/Chart.yaml (hunk 2 above)
Comment (rwaltr) [reply charts/stable/haste/Chart.yaml]: updated in https://github.com/k8s-at-home/charts/pull/800/commits/769f44f54a9f96ea280834e666a3c0e2d46314b0

**Comment by rwaltr** [comment]
preliminary testing looks good, and the helm chart is functional, but the image being hosted at `rwaltr/haste-server:2021` is not preferable, need to work to get this image working under `k8s-at-home's` repo

File: charts/stable/haste-server/Chart.yaml [hunk 11]
```diff
@@ -0,0 +1,23 @@
...
+sources:
+- https://github.com/seejohnrun/haste-server
+- https://github.com/rwaltr/haste-server
+maintainers:
+- name: rwaltr
+  email: rwalt@pm.me
+dependencies:
+- name: common
+  repository: https://library-charts.k8s-at-home.com
+  version: 2.3.0
```
Comment (onedr0p) [inline charts/stable/haste-server/Chart.yaml:23]: There was a release today that bumped this up to 2.3.1, mightneed to run helm docs one more time too after making this change.

File: charts/stable/haste-server/values.yaml [hunk 12]
```diff
@@ -0,0 +1,32 @@
...
+# IMPORTANT NOTE
+#
+# This chart inherits from our common library chart. You can check the default values/options here:
+# https://github.com/k8s-at-home/library-charts/tree/main/charts/stable/common/values.yaml
+#
+
+image:
+  repository: ghcr.io/rwaltr/haste-server
+  pullPolicy: IfNotPresent
+  tag: 2021-04-19
```
Comment (onedr0p) [inline charts/stable/haste-server/values.yaml:11]: Should we wait until the kah image is created?

**Review by onedr0p — CHANGES_REQUESTED** [review]

File: charts/stable/haste-server/values.yaml (hunk 12 above)
Comment (rwaltr) [reply charts/stable/haste-server/values.yaml:11]: We'll do it live

File: charts/stable/haste-server/Chart.yaml (hunk 11 above)
Comment (rwaltr) [reply charts/stable/haste-server/Chart.yaml:23]: bumpped

**Review by onedr0p — DISMISSED** [review]

**Review by onedr0p — APPROVED** [review]

**Comment by onedr0p** [comment]
what happened?

**Pull request merged**